src/styles/main.scss

```diff
@@ -1,3 +1,237 @@
-body {
+$color-main: #00acdc;
+$color-title: #060b35;
+$color-white: #fff;
+$color-text: #616070;
+$header-height: 60px;
+$card-margin: 16px;
+$logo-size: 40px;
+$star-size: 16px;
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+body,
+h2,
+p {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $color-text;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: $header-height;
+  background: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-img {
+  @include square($logo-size);
+}
+
+.nav {
+  display: block;
+
+  &__list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: stretch;
+    align-content: center;
+  }
+  
+  &__item {
+    display: flex;
+    margin-left: 20px;
+  }
+
+  &__link {
+    color: $color-title;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $header-height;
+    white-space: nowrap;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link:nth-of-type(4) {
+    flex-shrink: 0;
+  }
+
+  &__link:hover {
+    color: $color-main;
+  }
+
+  &__link:link {
+    position: relative;
+    text-decoration: none;
+  }
+
+  .is-active {
+    color: $color-main;
+  }
+
+  .is-active::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    border-radius: 2px;
+    background-color: $color-main;
+  }
+}
+
+.main-catalog {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
+  padding: 50px 40px;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.